### ISSUE TITLE
Feature Request: Ability to count entries

### ISSUE THREAD

**Issue by robynsmith** [maebert/jrnl#460]
It'd be cool if there was a way to count entries.

Something like:

```shell
jrnl -from today -count
10
```

I often count log entires by doing this:

```
cat file.log | wc -l
731
```

Just an idea. I'd partially consider actually implementing this if I have time. *shrug*

**Comment by philipsd6** [comment]
What about this?
```
jrnl -from today -s | wc -l
10
```
I'm a fan of the UNIX philosophy of one-thing-well, tools working together model. No need to add additional logic to `jrnl` just for counting when using `wc` to count the single-line summaries works as well.

**Comment by robynsmith** [comment]
I didn't know about `-s`.  It doesn't seem to do stuff for me? I tried it:

```shell
jrnl "Entry number one"
jrnl "Entry number two"
jrnl -from today -s

jrnl -from today -s | wc -l
1
```

It appears that whatever gets returned is just "blank"?

I tried this instead: `jrnl -from today --short`, which worked.

*shrug*

But you're right, since we can do it without adding a feature, I think we can probably close out this feature request! It might be a good idea to throw this in the docs though - is there a way of submitting a docs request on this repo?
